index.html

```diff
@@ -256,11 +256,12 @@ <h1>Get ready to Spot the Surveillance...</h1>
   </a-entity>
   <a-entity id="magic-carpet" mixin="exit">
   </a-entity>
-  <a-entity id="sky" geometry="primitive: sphere; radius:1000"  material="shader: flat; side: back;  src:#sts-sky" sound="src: #ambience; autoplay: false; loop: true; volume: 1;"  >
+  <a-entity id="sky" geometry="primitive: sphere; radius:1000"  material="shader: flat; side: back;  src:#sts-sky"  >
   </a-entity>
 </a-entity> <!-- end sts-live -->
 </a-scene>
 </div>
+<audio src="assets/audio/traffic-fade.mp3" controls="false" loop="true" autoplay="true" volume="0.1" ></audio>
```

Comment:
this should play only when the scene has been loaded and because of Chrome and Firefox's audio autoplay policies, the audio file needs to have `audioEl.play()` called on a user action (e.g., `click` of the "Enter VR" button).

cc @caseyyee. related PR #83